Playwright: Don't use instance variable for navigation retries

WunderBart opened the pull request:
A follow-up PR to https://github.com/Automattic/wp-calypso/pull/54987. Addresses @dpasque's comment where instance variable could get used up: https://github.com/Automattic/wp-calypso/pull/54987#discussion_r693198951.

WunderBart committed:
Don't use instance variable for navigation retries
Addresses https://github.com/Automattic/wp-calypso/pull/54987#discussion_r693198951

dpasque commented on packages/calypso-e2e/src/lib/components/sidebar-component.ts:
```diff
@@ -42,9 +40,10 @@ export class SidebarComponent {
 	 * @returns {Promise<void>} No return value.
 	 */
-	async navigate( item: string, subitem?: string ): Promise< void > {
+	async navigate( item: string, subitem?: string, _retries = 3 ): Promise< void > {
```
Complete nitpick here, but I think you can take off the `_` prefix here! I think that would match the surrounding style better. But that's take it or leave it! :)

WunderBart replied:
Removed in https://github.com/Automattic/wp-calypso/pull/55654/commits/aafc6117b196ad6c8a97444fee6fc36348d09230

dpasque reviewed (approved):
Looks great! The function argument approach is really nice. Added a tiny optional style nit.

🚢

WunderBart committed:
Remove the underscore prefix
As per https://github.com/Automattic/wp-calypso/pull/55654#discussion_r694070250
